Jenkinsfile

```diff
@@ -1,263 +1,22 @@
-                expression { "${PYTHON_DEPLOY_VERSION}" == "${PYTHON_VERSION}" }
-              }
-              stages {
-                stage("Build Docs") {
-                  steps {
-                    sh "${ACTIVATE} && make build-doc"
-                  }
-                }
-                stage("Build Package") {
-                  steps {
-                    sh "${ACTIVATE} && make build-package"
-                  }
-                }
-              } // stages within build and deploy
-            } // build and deploy stage
-        } // stages bracket within Python matrix
-        post {
-          always {
-            sh "${ACTIVATE} && make clean"
-            sh "rm -rf ${CONDA_ENV_PATH}"
-            // Generate a message to send to Slack.
-            script {
-              if (env.BRANCH == "main") {
-                channelName = "simsci-ci-status"
-              } else {
-                channelName = "simsci-ci-status-test"
-              }
-              // Run git command to get the author of the last commit
-              developerID = sh(
-                script: "git log -1 --pretty=format:'%an'",
-                returnStdout: true
-              ).trim()
-              slackID = githubUsernameToSlackName(developerID)
-              slackMessage = """
-                Job: *${env.JOB_NAME}*
-                Build number: #${env.BUILD_NUMBER}
-                Build status: *${currentBuild.result}*
-                Author: @${slackID}
-                Build details: <${env.BUILD_URL}/console|See in web console>
-            """.stripIndent()
-            }
-
-            // Delete the workspace directory.
-            deleteDir()
-          }
-          failure {
-            echo "This build triggered by ${developerID} failed on ${GIT_BRANCH}. Sending a failure message to Slack."
-            slackSend channel: "#${channelName}",
-                        message: slackMessage,
-                        teamDomain: "ihme",
-                        tokenCredentialId: "slack"
-          }
-          success {
-            script {
-              if (params.DEBUG) {
-                echo 'Debug is enabled. Sending a success message to Slack.'
-                slackSend channel: "#${channelName}",
-                          message: slackMessage,
-                          teamDomain: "ihme",
-                          tokenCredentialId: "slack"
-              } else {
-                echo 'Debug is not enabled. No success message will be sent to Slack.'
-              }
-            }
-          }
-        }
-      } // Python matrix bracket
-    } // Python matrix stage bracket
-  } // stages bracket
-} // pipeline bracket
+/*This file uses jenkins shared repo found at 
+https://github.com/ihmeuw/vivarium_build_utils
+The first line imports all modules with "_"
+The second line calls the standard pipeline from 
+the vars folder in the shared repo.
+Jenkins needs to be configured globally to use the correct branch
```

Comment:
Is this information still correct?

Reply:
It is. it could probably be cleaned up a tad but it's directing us to modify the setting here. This is where he said in slack that we need to change the branches if you want anything other than main since it appears that you can't parametrize this stuff.

<img width="581" alt="image" src="https://github.com/user-attachments/assets/8be70e3c-6aa6-44ff-ae34-80cfc15af8c6" />